config.json

```diff
@@ -467,6 +467,15 @@
       ]
     },
     {
+      "slug": "circular-buffer",
+      "difficulty": 8,
+      "topics": [
+        "Queues",
+        "Classes",
+        "Exceptions"
```

Comment:
I think it should be fine for people to add topics when they implement a new exercise if they want to, as long as they're valid topics as specified [here](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) :) What do you think @stkent?